hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/customrenderers/client/renderers/LinkRenderer.java

```diff
@@ -21,11 +24,21 @@
 public class LinkRenderer extends ButtonRenderer {
     @Override
     public void render(RendererCellReference cell, String text, Button button) {
-        button.setText(text);
+        Map<String, String> nameStatusMap = formatData(text);
+        final String targetName = nameStatusMap.get("name");
+        final String targetStatus = nameStatusMap.get("status");
+        button.setText(targetName);
         applystyle(button);
         // this is to allow the button to disappear, if the text is null
-        button.setVisible(text != null);
-        button.getElement().setId(new StringBuilder("link").append(".").append(text).toString());
+        button.setVisible(targetName != null);
+        button.getElement().setId(new StringBuilder("link").append(".").append(targetName).toString());
+        /*
+         * checking target Status for applying button style. If target status is
+         * not "CREATING", then the Rollout button is applying hyperlink style
+         */
+        final boolean isStatusCreate = targetStatus != null && "CREATING".equalsIgnoreCase(targetStatus);
```

Comment:
For `CREATING` a constant or even better the enum which defines the status can be used.